front/src/components/HeaderMenu/HeaderMenuDesktop.tsx

```diff
@@ -107,7 +107,7 @@ function HeaderMenuDesktopContent({ ...props }: IHeaderMenuProps) {
   );
 }
 
-function HeaderMenuDesktopContainer({ children }) {
+function HeaderMenuDesktopContainer({ children }: { children: React.ReactNode }) {
```

Comment:
I think children should be inferred correctly from all but most obscure Component types (or those self closing HTML tags). Is that not the case somehow?

Reply:
It's because they are React component props, that the linter requires explicit types even if they can be inferred, I believe.